app/services/proxy_status.rb

```diff
@@ -10,7 +10,3 @@ def self.find_status_by_humanized_name(name)
     status_registry.find { |s| s.humanized_name == name }
   end
 end
-require_dependency('proxy_status/version')
-require_dependency('proxy_status/tftp')
-require_dependency('proxy_status/puppet_ca')
-require_dependency('proxy_status/logs')
```

Comment:
If you lazy load the application, are they properly registered? I think that's one worry I had, which mostly affects development environments.

Reply:
We have eager loading in dev mode, but yeah, worth checking anyway...